.github/workflows/write-prerelease-comment.yml

```diff
@@ -33,7 +33,6 @@ jobs:
                   `\nWORKFLOW_RUN_PR_FOR_${packageName}=${match[2]}` +
                     `\nWORKFLOW_RUN_ID_FOR_${packageName}=${context.payload.workflow_run.id}`
                 );
-                break;
```

Comment:
as mentioned at the top of this script, this should be aligned with the content of `.github/extract-pr-and-workflow-id.js` but it was not (causing the C3 prerelease comment not to be created)